tasks/main.yml

```diff
@@ -7,15 +7,15 @@
 - name: "Generate Solr Master URL using variable"
   set_fact:
     solr_master_generated_url:
-      "{{ (solr_ssl_enabled == True) | ternary('https', 'http') }}://\
+      "{{ (solr_ssl_enabled) | ternary('https', 'http') }}://\
```

Comment:
May be [folded scalar](https://github.com/yaml/YAML2/wiki/Folded-Block-Scalars) will be better looking?